stable/README.md

```diff
@@ -1,167 +1,167 @@
-```
+Tiller is now available to all projects.
 
-We’ll check that the Helm client and Tiller server are able to communicate correctly by running helm version. The results should be as follows:
+## For Kubernetes
 
-```bash
-$ helm version
-Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
-```
+Start Helm and Tiller 
 
-You can also verify that the Tiller server is running via `kubectl`.
-```bash
-kubectl get pods -n kube-system
-NAME                                READY   STATUS    RESTARTS   AGE
-...
-tiller-deploy-8c5679674-k9c7m       1/1     Running   0          47m
-...
+```
+$ helm init --service-account tiller --tiller-namespace kube-system
 ```
 
+## For Red Hat OpenShift
 
-## Deploying NuoDB using Helm Charts
-
-### Grant OpenShift privileges
+Initialize the Helm client
 
-First, you should create a new namespace for NuoDB.
 ```bash
-oc new-project nuodb
+$ helm init --client-only
+$HELM_HOME has been configured at /home/clusteradmin/.helm.
+Not installing Tiller due to 'client-only' flag having been set
+Happy Helming!
 ```
 
-Create a new service account.
-```bash
-kubectl -n nuodb create serviceaccount nuodb
+Start Tiller 
+
+```
+helm init --service-account tiller --tiller-namespace kube-system
 ```
 
-Next, you will want to give your new service account the correct SecurityContextConstraints to run NuoDB.
-You can find the recommended SecurityContextConstraints in ([deploy/nuodb-scc.yaml](deploy/nuodb-scc.yaml)
+Assign the correct SecurityContextConstraints to the _nuodb_ and _default_ service accounts
 
 ```bash
 oc apply -f deploy/nuodb-scc.yaml -n nuodb
 oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:nuodb -n nuodb
 oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:default -n nuodb
 ```
 
-Here is a short list of NuoDB charts and their privilege requirements.
+## Confirm that the Helm client and Tiller server are able to communicate correctly. 
 
-| Charts | Privilege | Short Explanation |
-| ----- | ----------- | ------ |
-| transparent-hugepage| allowHostDirVolumePlugin: true | To mount hostPath and disable THP on host|
-| transparent-hugepage| volumes.hostPath | To mount hostPath and disable THP on host|
-| transparent-hugepage| seLinuxContext.* | To mount hostPath and disable THP on host|
-| admin, database| allowedCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
-| admin, database| defaultAddCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
+The results should be as follows:
 
+```bash
+$ helm version
+Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
```

Comment:
this version does not match the version you downloaded above

Reply:
good find. I didn't see the reference further down the page.